tests/README.md

```diff
@@ -54,62 +57,25 @@ To write an `istio.io` test, follow these steps:
    Fix the errors, if any, by updating the corresponding command in the `index.md` file and
    then regenerate the snips.
 
-1. Pick an appropriate location under the `tests/` directory for your new test.
-
-1. Create Go boilderplate that will invoke your test bash script using the following pattern:
-
-    ```golang
-    package <your-test-package>
-
-    import (
-        "testing"
-
-        "istio.io/istio/pkg/test/framework"
-
-        "istio.io/istio.io/pkg/test/istioio"
-    )
-
-    func Test<your-test>(t *testing.T) {
-        framework.
-            NewTest(t).
-            Run(istioio.NewBuilder("<your-test-name>").
-                Add(istioio.Script{
-                    Input: istioio.Path("scripts/<your-bash-script>.sh"),
-                }).
-                Defer(istioio.Script{
-                    Input: istioio.Inline{
-                        FileName: "cleanup.sh",
-                        Value: `
-    set +e # ignore cleanup errors
-    source ${REPO_ROOT}/content/en/docs/<your-snips-dir>/snips.sh
-    <your cleanup steps>`,
-                    },
-                }).
-                Build())
-    }
-    ```
-
-    NOTE: This Go boilerplate is a temporary requirement. It will not be needed in the future.
-    See https://docs.google.com/document/d/1r_NoxatNjzPsw0eXr6_9W0rqlkfelt7QfN2y5TF0yTo/edit#.
-
-1. Create your test bash script in the `scripts/` subdirectory.
+1. Create your test bash script `test.sh` next to the `snips.sh` you have just generated. Scripts that have other names will be ignored.
```

Comment:
#7501 is a WIP PR that is using making 3 different tests for the same doc/snip.sh. I'm not sure if that's a good idea (still need to see where that's going, and review), but if we decide it reasonable, maybe you should switch this back to running bash scripts with the suffix `test.sh` (your original design), instead of exactly `test.sh`. Most tests would still be just named `test.sh` but if somebody want's to split it into more than one bash script, they can have something like `part1_test.sh` and `part2_test.sh` in the same directory, and the framework will run both.

Reply:
Sounds reasonable. I'm allowing flexibility on the script name. How about stating it like this:
```suggestion
1. Create your test bash script `test.sh` next to the `snips.sh` you have just generated. The bash script name(s) should only have `test.sh` as their suffix. Scripts with other suffixes will be ignored.
```

Reply:
Here's my suggestion:
```
1. Create a test bash script named `test.sh` next to the `snips.sh` you have just generated.
   If your document is very large and you want to break it into multiple tests, create multiple scripts with the suffix `test.sh`, instead.
   Other scripts in the directory will be ignored.
```